src/ddmd/dsymbolsem.d

```diff
@@ -3633,7 +3634,7 @@ extern(C++) final class DsymbolSemanticVisitor : Visitor
                 error(tp.loc, "parameter '%s' multiply defined", tp.ident.toChars());
                 tempdecl.errors = true;
             }
-            if (!tp.semantic(paramscope, tempdecl.parameters))
+            if (!tp.tpsemantic(paramscope, tempdecl.parameters))
```

Comment:
Why the redundant `tp` in `tpsemantic`?

Reply:
Because `semantic` has too many pointless overloads making it hard to search for.